lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1993,6 +1993,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, the agent decorates logs with metadata to link to entities, hosts, traces, and spans.'
         },
+        :'code_level_metrics.enabled' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => true,
+          :description => 'If `true`, the agent will report source code level metrics for monitored methods.'
```

Comment:
I have a small concern about the word "monitored". Monitor mode for the agent and browser monitoring somewhat indicate all instrumented code. When I read this the first time, I expected code level metrics to be reported for every method the agent touches. 

What are your thoughts about using "traced" so that this leans more in the scope of `add_method_tracer`? 

My concern is about over-promising the scope of code level metrics in our documentation.

Reply:
Changed to "traced" with 6ff053b